radio/src/gui/480x272/layout.h

```diff
@@ -113,11 +113,12 @@ class BaseLayoutFactory: public LayoutFactory
     const ZoneOption * options;
 };
 
-#define MAX_REGISTERED_LAYOUTS 10
-
-extern unsigned int countRegisteredLayouts;
-extern const LayoutFactory * registeredLayouts[MAX_REGISTERED_LAYOUTS];
 Layout * loadLayout(const char * name, Layout::PersistentData * persistentData);
 void loadCustomScreens();
 
+#define MAX_REGISTERED_LAYOUTS          10
+extern unsigned int countRegisteredLayouts;
+#define registeredLayouts      getRegisteredLayouts()
```

Comment:
well I think if the compilation is OK you will be good.
if you have 5 minutes more I would even ask you if you could use std::list in themes / layouts so that there is only the singleton which is public